bindings/rust/generate/_generate_bindings.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+set -e -x
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+AWS_LC_DIR=$( cd -- "${SCRIPT_DIR}/../../../" &> /dev/null && pwd)
+GEN_DIR="${AWS_LC_DIR}"/bindings/rust/generate
+TMP_DIR="${AWS_LC_DIR}"/bindings/rust/tmp
+
+pushd "${GEN_DIR}"
+
+cargo clean
+cargo run "${TMP_DIR}"/aws-lc-sys
+cargo clean
+
```

Comment:
We should use the bindings generation process to verify the "bindings" feature of the crate we publish. The bindings we generate might always be in a filename of the form `<os>_<arch>_bindings.rs`.  This approach might remove some complexity.

Reply:
Put up a fix using feature flags for bindings pre-generation(`generate`) and using the fallback `binding.rs` generation(`native_bindings`). Not sure if it's doable, but I'm still looking to see if we can get the `os` and `arch` values during the preprocessing step, so that we can put it in the `use_bindings` macro.